docker-compose-dev.yaml

```diff
@@ -0,0 +1,27 @@
+services:
+  simulation:
+    extends:
+      file: docker-compose-base.yaml
+      service: simulation
+    ports:
+      - 7000:7000
```

Comment:
why not 8080 like every other service?

Reply:
In Development, so that I could start some services as well as the simulation. However, as soon as it is in the docker-infrastructure that does not matter anymore so I will change it